consensus/spos/consensusState.go

```diff
@@ -151,7 +152,7 @@ func (cns *ConsensusState) GetLeader() (string, error) {
 		return "", ErrEmptyConsensusGroup
 	}
 
-	return cns.consensusGroup[IndexOfLeaderInConsensusGroup], nil
```

Comment:
remove `IndexOfLeaderInConsensusGroup`? as it should not be used anymore, it can be done on next PR

Reply:
done in #6448